README.md

```diff
@@ -44,9 +44,11 @@ nope
 Dependencies: make, golang
 
 ```bash
-git clone https://github.com/direnv/direnv
-cd direnv
-make install
+go get github.com/direnv/direnv
+cd $GOPATH/src/github.com/direnv/direnv
+go get ./...
```

Comment:
this shouldn't be necessary since everything is in the vendor/ folder. what error did you get?

Reply:
If someone clones the repository and tried to build, they will fail. Same happens if they try to build directly using go build or if they try to use make to build.

This is because go language has some requirements regarding the location and layout of the source code.

Ideally the Makefile should take care of this and allow user to build regardless of the location where the direnv source code was cloned. Sadly it does no do this at this moment.